### ISSUE TITLE
PG7 for umbilical

### ISSUE THREAD

**Issue by nmr94** [PhilBaz/-Super-Stealth-Orbiter-Orbiter-1.5-#2]
I have an umbilical cord instead of a cable chain, any chance you could add an update to allow PG7 cable glands to be used with this mod? I’d love to use it.

thanks!

**Comment by PhilBaz** [comment]
Yes, Actually I have a whole set of mods for this...CANBUS, umbilical. A version with square nuts instead of insets for a few key bolts. Give me a few days Ill get it up on Printable. Been meaning to do that forever.

**Comment by nmr94** [comment]
That would be amazing, I've got an umbilical at the moment so it's the only thing stopping me using your mod. No rush in getting it up on here, but if you could drop a comment here when it's up, I should get an email and I'll get it installed.

Thanks for your hard work!

**Comment by nmr94** [comment]
Did you ever manage to get these up? :)  Took a look over at printables but didn't see anything.

**Comment by PhilBaz** [comment]
@nmr94 Soon! just getting the house set up...give me a few more days and ill make it happen, promise. Ill post the whole thing.

**Comment by PhilBaz** [comment]
@nmr94 going to post it tonight...maybe 30 min. Sorry for the delay. Hope you enjoy it. Its a good little sytem, been working great for me.

**Comment by PhilBaz** [comment]
@nmr94 https://www.printables.com/model/1070522-super-stealth-orbiter-canbus-umbilical-for-voron-s
